lib/inventory_refresh/graph/topological_sort.rb

```diff
@@ -45,14 +45,14 @@ def topological_sort
           max_depth -= 1
           if max_depth <= 0
             message = "Max depth reached while doing topological sort, your graph probably contains a cycle"
-            #logger.error("#{message}:\n#{graph.to_graphviz}")
+            # logger.error("#{message}:\n#{graph.to_graphviz}")
```

Comment:
Not sure why this was left commented out in the first place and not simply deleted